chapter5.tex

```diff
@@ -40,6 +40,49 @@ \subsection{Moc AI}
 
 \section{Dalsze perspektywy rozwoju projektu}
 
+Aplikacja do gry w~brydża została stworzona głównie
+na potrzeby asystenta, aby można było go wykorzystać w~grze
+poprzez sieć Internet. Tworzona była w~technologiach, które
+w~naszej opinii są popularne, a~ich rozwój jest stale wspierany.

```

Comment:
Nie tylko w naszej opinii, tylko ogólnie są popularne.

Reply:
Jeżeli są popularne musimy to udowodnić, stosując stwierdzenie "w naszej opinii" nie musimy takiego dowodu przedstawiać.

Reply:
Nie podawaliśmy stack overflow survey?

Reply:
Tak, ale dla konkretnego frameworku. Tu mówimy tak naprawdę o wszystkich technologiach i o tym, że się rozwijają na co też dowód by się przydał. Własna opinia jest najbezpieczniejsza. tym bardziej w podsumowaniu.